kvwrapper_etcd/kvwrapper_etcd.go

```diff
@@ -32,7 +43,11 @@ func (e EtcdWrapper) Set(key string, val string, ttl uint64) error {
 
 // GetVal returns a single KeyValue found at key
 func (e EtcdWrapper) GetVal(key string) (*kvwrapper.KeyValue, error) {
-	r, err := e.client.Get(key, false, true)
+	options := &etcd.GetOptions{
+		Sort:      false,
+		Recursive: true,
```

Comment:
Shouldn't this be false ? Caus we want only the value residing at that key and not all its children.